Mailer/app/api/v1/Handlers/Mailers/MailLogsHandler.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Remp\MailerModule\Api\v1\Handlers\Mailers;
+
+use Nette\Utils\Json;
+use Nette\Utils\JsonException;
+use Remp\MailerModule\Repository\ListsRepository;
+use Remp\MailerModule\Repository\LogsRepository;
+use Remp\MailerModule\Repository\MailTypesRepository;
+use Tomaj\NetteApi\Handlers\BaseHandler;
+use Tomaj\NetteApi\Params\InputParam;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+use Tracy\Debugger;
+
+class MailLogsHandler extends BaseHandler
+{
+    private $logsRepository;
+
+    public function __construct(
+        LogsRepository $logsRepository
+    ) {
+        parent::__construct();
+        $this->logsRepository = $logsRepository;
+    }
+
+    public function params()
+    {
+        return [
+            new InputParam(InputParam::TYPE_POST, 'where', InputParam::OPTIONAL),
```

Comment:
Again, we cannot accept `where`, we need to accept specific parameters and only allowed set/types of values.